initial-promises/node-server/common/1.0/site.cf

```diff
@@ -44,78 +44,95 @@ bundle common g
       "rudder_ncf"                 string => "${rudder_base}\ncf";
       "escaped_workdir"            string => escape("${sys.workdir}");
-      "rudder_curl"                string => "${rudder_base_sbin}\curl\curl.exe";
-
+      "rudder_curl"                string => "\"${rudder_base_sbin}\curl\curl.exe\"";
       "uuid_file"                  string => "${rudder_base}\etc\uuid.hive";
       "rudder_disable_agent_file"  string => "${rudder_base}\etc\disable-agent";
 
     aix::
 
-      "rudder_gzip"                       string => "/usr/bin/gzip";
-      "inventory_upload_protocol"         string => "http";
+      "gzip"                       string => "/usr/bin/gzip";
+      "inventory_upload_protocol"  string => "http";
 
     !aix::
 
-      "inventory_upload_protocol"         string => "https";
+      "inventory_upload_protocol"  string => "https";
 
     !aix.!windows::
 
-      "rudder_gzip"                       string => "/bin/gzip";
+      "gzip"                       string => "/bin/gzip";
 
     any::
-      "uuid"                              string => readfile("${g.uuid_file}", 60);
-      "server_shares_folder"              string => "/var/rudder/share/${uuid}/promises/shares";
-      "rudder_var_reports"                string => "${rudder_var}/reports";
-      "davuser"                           string => "rudder";
-      "davpw"                             string => "rudder";
-      "minicurl"                          string => "${rudder_bin}/rudder-perl ${sys.workdir}/inputs/common/utilities/minicurl";
-      "excludedreps"                       slist => { "\.X11", ".*kde.*", "\.svn", "perl" };
-       "ncf_local_excludedreps"            slist => { "50_techniques", @{g.excludedreps}};
-      "rudder_tools_origin"               string => "/var/rudder/tools";
-      "rudder_tools_updated_origin"       string => "${rudder_tools_origin}/rudder_tools_updated";
-      "rudder_tools_updated"              string => "${rudder_tools}/rudder_tools_updated";
-
-      "rudder_ncf_common_origin"          string => "/var/rudder/ncf/common";
-      "rudder_ncf_origin_common"          string => "/usr/share/ncf/tree";
-      "rudder_ncf_origin_local"           string => "/var/rudder/configuration-repository/ncf";
-
-      "uncompressed_inventory_file_types"  slist => { ".*.ocs" };
-      "inventory_file_types"               slist => { "@{uncompressed_inventory_file_types}", ".*.gz" };
-      "signed_inventory_file_types"        slist => { "@{inventory_file_types}", ".*\.sign" };
-
-      "server_roles_path"                 string => "${rudder_base}/etc/server-roles.d/";
+      "uuid"                        string => readfile("${g.uuid_file}", 60);
+      "server_shares_folder"        string => "/var/rudder/share/${uuid}/promises/shares";
+      "rudder_var_reports"          string => "${rudder_var}/reports";
+      "davuser"                     string => "";
+      "davpw"                       string => "";
```

Comment:
Actually I see they are in the variables.json file, but for some reason they have not appeared here.